packages/rx-utils/src/create-handler.utils.ts

```diff
@@ -1,18 +1,23 @@
 import { Observable, Subject } from 'rxjs';
 
-export interface THandle<A> {
-	(): void;
-	(a: A): void;
+interface HandlerVoid {
+	readonly value$: Observable<void>;
+	readonly handle: (a?: void) => void;
```

Comment:
What does `(a?: void)` control and what for?

Reply:
to save possibility to pass undefined as an argument. a little compatibility with the old code. but if it is a breaking change we could force not to pass anything.

Reply:
Yeah let's keep things clean

Reply:
amended it